### PR TITLE
Error page

### PR DESCRIPTION
**Pull request by waweewawee**

### CODE REVIEW

File: client/src/components/ErrorPage.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import {
```
Comment (GeneDrive) [inline client/src/components/ErrorPage.tsx:1]: this would be better in one line I think

File: client/src/components/ErrorPage.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+						}}>
+                    Pagina niet gevonden
+					</Typography>
+					<Typography sx={{
+						py:3
+					}}>
+                    Error 404: De pagina die u probeert te bereiken is niet gevonden
+					</Typography>
+					<Card>
+						<CardActionArea onClick={()=>navigate('/')}>
```
Comment (GeneDrive) [inline client/src/components/ErrorPage.tsx:50]: onclick on button instead of card

**Review by GeneDrive — CHANGES_REQUESTED** [review]

File: client/src/components/ErrorPage.tsx (hunk 1 above)
Comment (ManoloTonto1) [reply client/src/components/ErrorPage.tsx:1]: No

**Review by GeneDrive — APPROVED** [review]
looking hot

**Comment by waweewawee** [comment]
Error page toegevoegd en routing van error page naar homepage functionaliteit werkt.

**Comment by waweewawee** [comment]
closed #43

**Pull request merged**